### PR TITLE
Fixes being able to insert hand emote items (kisses, slappers, etc) into storage

### LINKED ISSUE #65168: Circle/slapper in evidence bag.
## Round ID:

[Round ID]: Terry #179041

## Testmerges:

It wouldnt let me view them, but im certain it wasnt any of those.

## Reproduction:
1. Get evidence bag
2. Put slapper/circle in bag (*slap/*circle)
3. Ascend by putting a slapper or circle in a evidence bag
4. Take them out, notice that the bag is now unusable
5. Game is unplayable

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Makes circlehands and slappers huge, just like offhand is.
I dunno if it is worth adding a check to evidence bag for dropdel to not fit, because I don't know how much the flag is used, but I thought changing the hand's weights would be better in the long run for those specific items since this problem might not be limited to just evidence bags.

## Why It's Good For The Game

Closes https://github.com/tgstation/tgstation/issues/65168 and possibly more edgecases.

## Changelog

:cl:
fix: Slappers and circlehands don't fit in storage bags anymore.
/:cl:

### CODE REVIEW

**Comment by ArcaneMusic** [comment]
As an additional safeguard, you should have evidence bags check of the object you're trying to add is abstract.

**Comment by JohnFulpWillard** [comment]
> As an additional safeguard, you should have evidence bags check if the object you're trying to add is abstract.

image

**Review by Ghommie — CHANGES_REQUESTED** [review]
Could you please use the `TRAIT_NO_STORAGE_INSERT` trait instead? Changing the `w_class` is a suboptimal solution because it won't stop the item from being inserted in a _hypothetical_ (unless I'm wrong, huge items cannot be inserted in the BoH anymore) storage item with a high enough `max_w_class`.

**Review by MrMelbert — CHANGES_REQUESTED** [review]
What Ghommie said, changing w_class is a bandaid 

`HAND_ITEM` flagged items probably shouldn't exist in a state which it's not held by a mob, anyways

File: code/game/objects/items/hand_items.dm [hunk 1]
```diff
@@ -416,21 +415,19 @@
...
+/obj/item/hand_item/kisser
 	name = "kiss"
 	desc = "I want you all to know, everyone and anyone, to seal it with a kiss."
 	icon = 'icons/mob/animal.dmi'
 	icon_state = "heart"
 	inhand_icon_state = "nothing"
-	force = 0
-	throwforce = 0
-	item_flags = DROPDEL | ABSTRACT | HAND_ITEM
+	w_class = WEIGHT_CLASS_HUGE
```
Comment (MrMelbert) [inline code/game/objects/items/hand_items.dm:424]: Leftover?

Comment (JohnFulpWillard) [reply code/game/objects/items/hand_items.dm:424]: yeah my bad

**Pull request merged**